js/src/services/vue.service.js

```diff
@@ -112,6 +150,24 @@ class VueService {
     getVue() {
         return Vue;
     }
+
+    /*
+    * Mark a component as loaded.
+    */
+    componentIsLoaded(name) {
```

Comment:
`markComponentLoaded` - this is not getter

Reply:
I will change